pyne/pnnl/materials_compendium.py

```diff
@@ -0,0 +1,411 @@
+        NonIsotopic (bool): Whether the element is non-isotopic.
+        Element (str): The name or identifier of the element.
+        WeightFraction (float): Weight fraction of the element.
+        AtomicMass (float): Atomic mass of the element.
+        ZAID (str): ZAID (unique identifier) of the element.
+        AtomFraction (float): Atom fraction of the element.
+        AtomDensity_whole (float): Atom density of the element as a whole number.
+        AtomFraction_whole (float): Atom fraction of the element as a whole number.
+        id (str): Identifier of the element.
+        Isotopes (List[Isotope]): List of Isotope objects representing the isotopes of the element.
+        AtomDensity (float): Atom density of the element.
+        AtomicMass_whole (float): Atomic mass of the element as a whole number.
+        Abundances (str): Abundances of the element.
+    """
+
+    WeightFraction_whole: float
+    NonIsotopic: bool
+    Element: str
+    WeightFraction: float
+    AtomicMass: float
+    ZAID: str
+    AtomFraction: float
+    AtomDensity_whole: float
+    AtomFraction_whole: float
+    id: str
+    Isotopes: List[Isotope]
+    AtomDensity: float
+    AtomicMass_whole: float
+    Abundances: str
+
+    @staticmethod
+    def from_dict(obj: Any) -> "Element":
+        """
+        Create an Element object from a dictionary representation.
+
+        Parameters:
+            obj (dict): A dictionary containing attributes of the Element.
+
+        Returns:
+            Element: A new Element object with attributes initialized using the values from the dictionary.
+        """
+        _WeightFraction_whole = float(obj.get("WeightFraction_whole"))
+        _NonIsotopic = bool(obj.get("NonIsotopic"))
+        _Element = str(obj.get("Element"))
+        _WeightFraction = float(obj.get("WeightFraction"))
+        _AtomicMass = float(obj.get("AtomicMass"))
+        _ZAID = str(obj.get("ZAID"))
+        _AtomFraction = float(obj.get("AtomFraction"))
+        _AtomDensity_whole = float(obj.get("AtomDensity_whole"))
+        _AtomFraction_whole = float(obj.get("AtomFraction_whole"))
+        _id = str(obj.get("id"))
+        _Isotopes = [Isotope.from_dict(y) for y in obj.get("Isotopes")]
+        _AtomDensity = float(obj.get("AtomDensity"))
+        _AtomicMass_whole = float(obj.get("AtomicMass_whole"))
+        _Abundances = str(obj.get("Abundances"))
+        return Element(
+            _WeightFraction_whole,
+            _NonIsotopic,
+            _Element,
+            _WeightFraction,
+            _AtomicMass,
+            _ZAID,
+            _AtomFraction,
+            _AtomDensity_whole,
+            _AtomFraction_whole,
+            _id,
+            _Isotopes,
+            _AtomDensity,
+            _AtomicMass_whole,
+            _Abundances,
+        )
+
+
+@dataclass
+class Mol:
+    """
+    Represents a molecular entity with various attributes related to its properties.
```

Comment:
I don't understand how this data can represent a "molecular entity"?

Reply:
Mols is a list object containing json data Mols, Element and Isotope

```json
            "Mols": [
                {
                    "Mols": 12,
                    "Element": "C",
                    "Isotope": "C12"
                },
                {
                    "Mols": 23,
                    "Element": "H",
                    "Isotope": "H23"
                }
            ],
```